scripts/release/release-utils.js

```diff
@@ -50,19 +50,6 @@ function wrapInLi(content) {
     return `<li>${content}</li>`;
 }
 
-/**
- * Provide the source project.pbxproj for iOS and macOS and get it updated with the BSK reference
- * @param {string} projectPbxprojContent
- * @param {string} commit
- * @returns {string}
- */
-function updateProjectPbxproj(projectPbxprojContent, commit) {
-    const bskPackageRegex =
-        /(repositoryURL = "https:\/\/github\.com\/duckduckgo\/BrowserServicesKit";\s+requirement = {\s+kind = )(exactVersion)(;\s+)(version = \d+.\d+.\d+;)/i;
-
-    return replaceInString(projectPbxprojContent, bskPackageRegex, `$1revision$3revision = ${commit};`);
-}
```

Comment:
No longer needed as Autofill updates will only ever go through a BSK reference update, there will no longer be any remote reference to BSK that needs updating. 🎉